### ISSUE TITLE
No warning for unused delegate

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#43113]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity.visualstudio.com/content/idea/971269/no-warning-for-unused-delegate.html)._

---
Visual Studio does not issue a warning for unused delegates like it does for other unused fields/variables.

```
private delegate bool RunTestDelegate(TEST name);  // Unused delegate, no compiler warning
```

---
#### Visual Studio Feedback System on 3/31/2020, 11:35 PM: 

Thank you for taking the time to provide your suggestion.  We will do some preliminary checks to make sure we can proceed further.  We'll provide an update once the issue has been triaged by the product team.
